module/mailnews/mailfile.c

```diff
@@ -134,6 +134,7 @@ void MailNewMessage(int server_index, char *sender, int num_recipients,
    char *subject = "";
    char *ptr;
    int i, num;
+   int subject_id_index = 1;
```

Comment:
Are you sure this should be 1 and not 0?  I think that means that the default in error cases will be German.